Source/Scene/Expression.js

```diff
@@ -541,27 +541,39 @@ define([
 
     function parseKeywordsAndVariables(ast) {
         if (isVariable(ast.name)) {
-            return new Node(ExpressionNodeType.VARIABLE, getPropertyName(ast.name));
+            var name = getPropertyName(ast.name);
+            if (name.substr(0, 8) === 'tiles3d_') {
```

Comment:
It's already here: https://github.com/AnalyticalGraphicsInc/3d-tiles/tree/master/Styling#built-in-variables, unless that should be expanded on.